### PR TITLE
Add a note about migrating to a FIPS machine.

### PR DESCRIPTION
**Pull request by goosemania**
closes #7992
https://pulp.plan.io/issues/7992

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -77,6 +77,15 @@ the migration is done and remove those two checksum types from the allowed list.
 the setting, you likely will need to run ``pulpcore-manager handle-artifact-checksums`` to remove
 unsupported checksums from the database, or Pulp will refuse to start.
 
+.. note::
+
+    If you'd like to run Pulp 3 on a machine where FIPS is enabled, you would need to patch
```
Comment (ggainey) [inline docs/configuration.rst:82]: ^would^will

File: docs/configuration.rst [hunk 2]
```diff
@@ -77,6 +77,15 @@ the migration is done and remove those two checksum types from the allowed list.
...
+    Django and Pulp at your own risk https://hackmd.io/@pulp/Pulp3-FIPS-matrix#Patches. The
+    patches will go away when Django project starts supporting FIPS mode.
```
Comment (ggainey) [inline docs/configuration.rst:84]: "...starts supporting FIPS mode (see https://code.djangoproject.com/ticket/28401 for details)."

**Review by ggainey — CHANGES_REQUESTED** [review]
Just some minor wordsmithing!

**Comment by goosemania** [comment]
@ggainey , ready for re-review

**Review by ggainey — APPROVED** [review]
lg2m!

**Pull request merged**